docker-compose.yml

```diff
@@ -6,15 +6,8 @@ services:
     build:
       context: .
       dockerfile: ./docker/setup.Dockerfile
-    environment:
-      - DATABASE=${MARIADB_DATABASE:-darkflame}
-      - DATABASE_HOST=database
-      - DATABASE_USER=${MARIADB_USER:-darkflame}
-      - DATABASE_PASSWORD=${MARIADB_PASSWORD:-darkflame}
-      - EXTERNAL_IP=${EXTERNAL_IP:-darkflame}
```

Comment:
These changes seem unrelated?

Reply:
These changes were required to perform variable substitution in the `.ini` files. Previously the `.ini` files were copied from the main repository to the _setup_ service which then used these variables to update the `.ini` files which were then shared through a volume with the _darkflame_ (server) service. The `.ini` files prepared during CMake build were simply discarded.

Therefore I have moved all these environment variables to the _darkflame_ service. Otherwise the original `.ini` files without substituted variables would be used.